### ISSUE TITLE
Put GUI code in a module

### ISSUE THREAD

**Issue by Jookia** [Jookia/BGJ2024#3]
A lot of the GUI code is accessibility-specific and shouldn't be touched that often without testing. It should be put in its own module.

**Comment by Jookia** [comment]
Closed by #9
